docs/standard/glossary.md

```diff
@@ -35,6 +35,12 @@ A cross-platform, high-performance, open source implementation of ASP.NET built
 
 See [ASP.NET Core](/aspnet/#pivot=core).
 
+## assembly
+
+A *.dll* file that contains a collection of APIs that can be called by apps or other assemblies.
```

Comment:
I see that you already added. We even have that in the CLI commands reference.

https://github.com/dotnet/docs/search?l=Markdown&q=binaries&type=&utf8=%E2%9C%93